src/Optimization.js

```diff
@@ -24,4 +25,38 @@ module.exports = class extends ChainedMap {
       'usedExports',
     ]);
   }
+
+  minimizer(name) {
+    if (!this.minimizers.has(name)) {
+      this.minimizers.set(name, new Plugin(this, name));
+    }
+
+    return this.minimizers.get(name);
+  }
+
+  toConfig() {
```

Comment:
I think `Config.js` will need it's `toConfig()` changing too, so that it uses the new `toConfig()` here?

(And we need a new test to catch this; presumably in `test/Config.js`?)